src/main/java/com/commercetools/paymenttoorderprocessor/jobs/actions/OrderCreator.java

```diff
@@ -106,14 +107,26 @@ private void sendRequestToCreateOrder(CartAndMessage cartAndMessage) {
             httpResponse = httpClient.execute(httpRequest).toCompletableFuture()
                     .get(createOrderTimeout, TimeUnit.MILLISECONDS);
 
+            String responseBody = getStringFromResponseBody(httpResponse.getResponseBody());
+
             if (httpResponse.hasSuccessResponseCode()) {
+                if (ObjectUtils.compare(httpResponse.getStatusCode(), HttpStatusCode.CREATED_201) == 0) {
+                    // normal case: cart is created successfully
+                    LOG.info("Success with status {}. Processed cart id=[{}], created order id=[{}]",
+                            httpResponse.getStatusCode(), cartAndMessage.getCart().getId(), responseBody);
+                } else {
+                    // the request is finished successfully, but this cart can't be processed.
+                    // this case should be reported, but not re-tried any more
+                    LOG.warn("Request is successful with status {}, but order is not created. "
```

Comment:
also a shop may have a lot of other logs thus more difficult to find proper log after couple of hours/days. In our case we keep the error/warning message in place where it is found.